WordPress/Classes/ViewRelated/Reader/Tab Navigation/ReaderTabView.swift

```diff
@@ -72,6 +69,19 @@ extension ReaderTabView {
         tabBar.tabBarHeight = Appearance.barHeight
         WPStyleGuide.configureFilterTabBar(tabBar)
         tabBar.addTarget(self, action: #selector(selectedTabDidChange(_:)), for: .valueChanged)
+
+        viewModel.fetchReaderMenu() { [weak self] sections in
+            guard let sections = sections, let self = self else {
+                return
+            }
+            UIView.animate(withDuration: Appearance.tabBarAnimationsDuration) {
```

Comment:
we could, but I would still use the inner scope of the `fetchReaderMenu()` closure (instead of having the convenience method to point to the outer scope). To do so, either I pass three parameters (self.tabBar, self.buttonsStackView and sections) or the inner reference to `self` (aka `ReaderTabView`) along with the sections, and both seem a little overkill for a few lines of code.